puppetboard/templates/report.html

```diff
@@ -48,26 +48,62 @@
       <td>{{event.item['old']}}</td>
       <td>{{event.item['new']}}</td>
     </tr>
-    {#    <tr>
-      <td class='message' colspan='4'>
-        <div id='message-event-{{loop.index}}'>
-          {{event.item['message']}}
-        </div>
-      </td>
-    </tr>#}
+    {% endfor %}
+  </tbody>
+</table>
+
+<h1>Logs</h1>
+<table class="ui basic table compact">
+  <thead>
+    <tr>
+      <th>Timestamp</th>
+      <th>Source</th>
+      <th>Tags</th>
+      <th>Message</th>
+      <th>Location</th>
```

Comment:
The source, tags and location were inspired by @raphink 's work on https://github.com/puppet-community/puppetboard/pull/166 but with slight modifications